eva-accession-import/src/main/resources/application.properties

```diff
@@ -0,0 +1,37 @@
+parameters.assemblyAccession=
+parameters.assemblyName=
+parameters.assemblyReportUrl=
+parameters.taxonomyAccession=
+parameters.chunkSize=100
+#parameters.pageSize=
+parameters.forceRestart=false
+parameters.fasta=
+
+spring.batch.job.names=IMPORT_DBSNP_VARIANTS_JOB
```

Comment:
It would be useful to indicate that this variants have coordinates, for instance `import-dbsnp-mapped-variants-job`. Please check but I think that lowercase and dashes is the convention we have followed in other projects to name jobs.